src/client/src/orangehrmPimPlugin/components/ReportCriterionRange.vue

```diff
@@ -95,7 +97,7 @@ export default {
         v => {
           return (
             parseInt(v) < parseInt(props.valueY) ||
-            this.$t('general.should_be_less_than_upper_bound')
+            translate('general.should_be_less_than_upper_bound')
```

Comment:
I fixed this in #1255 please rebase from 5.0
https://github.com/orangehrm/orangehrm/pull/1255/commits/55765535748072b51229b443a317c47c950da3c9